### DISCUSSION TITLE
Tokens stuck using Wormhole (SOL to ETH)

### DISCUSSION THREAD

**Discussion by ghost** [wormhole-foundation/wormhole#447] [Problems / bugs] [answered]
The transaction shows as success sending wLuna from SOL to ETH; however, it failed at the last step. Here is the solana transaction signature: 

65P7myte3qXddxBdGFbCEzsWqvPUbozmv8tPou4tMMqh6i7JHL746yf5uRHWvVzsVDceB7p6zYbT5G2MtHC1tpqs

Can you help get the VAA information so i can resubmit in my wallet?

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading https://v1.wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If v1.wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

Note that Wormhole v2 was recently launched on wormholebridge.com, which fixes the bug you encountered! You will still have to use v1 to complete your in-progress transfer.

You can also try using the third-party https://www.interstellar.tk site to retry your VAA submission.

---

Digest (for reference only): 6fefd34e9f730584359537fb06a76c70ea3b56d0e3ecc36eddabcd0deda5ae7d

Signed VAA bytes to submit to the contract:

> 0x01000000010d0077a139d48e8fdbc2ac8ff57a93cac501bbf584f6accf9e6ea8a546b8b47847ee2bac584ce73bd72e81bb901ca34eb5de2b7f84e791f9d9b66d558de7664e866b00015a18c4fc17d1900247dc1eb73d9441eb7d884ba2a8ab4da78b7e5dcbc203c139690a6b4f8de00356760bb74e69c3646ffc0991fb5c20c632a5aecf479ae9a6e100022bde7490a007d8ae808e7a49107f75cb4eab1c79d244d4aa560b20b1cdcc186f6f7244a67376b25ea96f220317c48860406b99b29c24335bd9e47c84c6edf9f80103abce70d254acfb51784f3bde8b2ec549473ea1050e1cbf35b8674ec64f13c3232e7e677251b3c41319c28dcc4c99c43ace14417cd88a32d1954b9fea80b1e8d4010554c7bc50cf38c7b0cf8d4d8272b0f83ae4e62b4ebb2b07821dd903fe7a3dd0814aa5e624e17407b60b19f852afaa79d485ecdada51b45a0a934152022926b5f201060f92b91fc56a602800050cf69156a32aef2ed5f3e61141c8c38ec87722e3247e4062841177fc30680fb3b4f061d66ed445577cf43b451c3ad54db9c57c465cca0007f7efe213eca01168fb91d98f86e5e09b7239a6e050969e4ff09f5fb0f982afc4669f3b5db6aae666d5116684aa23b43fc8a33a5309e18e15ef4dfcc4e33f78e10108b4043d7bc7f229d59bb6d56c597f8e38ee146f3de4e3cb11e39d98ffbc318e307788d5831b98ea5d6375ae2c303a06b18b3645e4d76087c557564a89fad8c683000abc40860e170fee70e3422fddb6ba0fa87835036b536eb6d207a26ea8116eeb153bb5609c47ace3cd8fc59dc3b9b2abb30606d4d9071801424e6e889ec6e9cf24010cff0a005db4e02f595f0e31fdf2bbdf769ffd9d8bbca71b1ee96a638152600f994a8c74c567bb98f0863ad8515fb2f8abfbc076c6f74e551730ed36f3a1d281f3010d01a9035c75a9eda2302b39fae712a1a82980972dfc326cabb185c65745faab513dc3b4007bbc2bfaeb49b658cdcaf3165b8d9bd1fdc34b961b89760d82eec870011145d2f3056a55cb2948a8f56959fec4586d39604604dd4bb3f59d6e111da3a6ff10671418be319e65ecbc1351a878bf7f2bb2538be7f1666b0b5c10dc8c09926c00128f1d62f8d2476b8ac57a56000741dbc02001c46bc986c9babb722504bad433c30a21e563e20aa5721f6b29380e9aeb9e44b365caf2656fa6cc40260f373a32bf016143ff7e1000014d870102672c3058583352d1664ad64db6d9083ce3d47f06a192392e0bad84736c278354000000000000000000000000e6375df92796f95394a276e0ba4efc4176d41d4902000000000000000000000000d2877702675e6ceb975b4a1dff9fb7baf4c91ea90900000000000000000000000000000000000000000000000000000289514db818
